frontend/src/LoginPage/LoginPage.jsx

```diff
@@ -46,23 +46,23 @@ class LoginPage extends React.Component {
       return;
     }
 
-    authenticationService.login(email, password).then(this.authSuccessHandler, this.authFailureHandler);
-  };
-
-  authSuccessHandler = () => {
-    const params = queryString.parse(this.props.location.search);
-    const { from } = params.redirectTo ? { from: { pathname: params.redirectTo } } : { from: { pathname: '/' } };
-    this.props.history.push(from);
-    this.setState({ isLoading: false });
-  };
-
-  authFailureHandler = () => {
-    toast.error('Invalid email or password', {
-      toastId: 'toast-login-auth-error',
-      hideProgressBar: true,
-      position: 'top-center',
-    });
-    this.setState({ isLoading: false });
+    authenticationService.login(email, password).then(
```

Comment:
We need `this.authSuccessHandler` and `this.authFailureHandler` as they are used by `GoogleSSOLoginButton` here: https://github.com/ToolJet/ToolJet/blob/709d3e3df4060f2ee193624e9f8477a442f92af2/frontend/src/LoginPage/LoginPage.jsx#L136-L139

Reply:
This refactoring was done for this purpose. Can we make do it in such a way that this refactoring is kept intact?

Reply:
This refactoring was only present in the `ee` repo, that is why this conflict occured.

Reply:
owh okay, sure 👍🏻